app/org/sagebionetworks/bridge/models/surveys/BloodPressureConstraints.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import static org.sagebionetworks.bridge.models.surveys.UIHint.BLOODPRESSURE;
+
+import java.util.EnumSet;
+
+public class BloodPressureConstraints extends Constraints {
+    public BloodPressureConstraints() {
+        setDataType(DataType.DECIMAL);
+        setSupportedHints(EnumSet.of(BLOODPRESSURE));
```

Comment:
Blood pressure is a number like 120/80 (possibly with an optional pulse value, it's not a decimal type. It might be its own data type, or maybe a string in a set format like X/Y Z